home.html

```diff
@@ -0,0 +1,205 @@
+        <div class="line"></div>  
+        <div class="row py-5 gap-1" id="speaker-container"></div>
+        <div class="see-more">
+          <button class="more-btn fs-5 py-1">More <img src="Images/arrow_down.png" alt="See more"></button>
+        </div>
+        <div class="see-less d-none">
+          <button class="more-btn fs-5 py-1">Less <img src="Images/arrow_down.png" alt="See less" class="flip-img"></button>
+        </div>      
+      </section>
+
+      <section class="sponsors container-fluid text-center pt-3">
+        <h2 class="fs-5 text-light ">Sponsors & Partners</h2>
+        <div class="line mt-2 mb-4"></div> 
+        <div class="all-sponsors container-fluid row overflow-auto py-4">
+          <div class="col sponsor1 w3-animate-left"><img src="Images/sponsor_3.png" alt="United Nations"></div>
+          <div class="col sponsor1 w3-animate-left"><img src="Images/sponsor_4.png" alt="Forbes"></div>      
+          <div class="col sponsor1 w3-animate-left"><img src="Images/sponsor_2.png" alt="Coca-cola company"></div>
+          <div class="col sponsor1 w3-animate-left"><img src="Images/sponsor_7.jpeg" alt="Glam"></div>
+          <div class="col sponsor1 w3-animate-left"><img src="Images/sponsor_ 17.png" alt="Reebok"></div>
+          <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_1.png" alt="Obama Foundation"></div>     
+          <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_8.png" alt="Amazon"></div>
+          <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_5.png" alt="Samsung"></div>
+          <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_ 11.png" alt="sprite"></div>
+          <div class="col sponsor2 d-md-none w3-animate-left"><img src="Images/sponsor_ 13.png" alt="Samsung"></div>
+          <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_6.png" alt="Fenty beauty"></div>     
+          <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_9.png" alt="Microsoft"></div>
+          <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_10.png" alt="IBM"></div>
+          <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_ 14.png" alt="IBM"></div>
+          <div class="col sponsor3 d-md-none w3-animate-left"><img src="Images/sponsor_1.png" alt="IBM"></div>
+          <a class="prev text-decoration-none d-none d-md-block">❮</a>
+          <a class="next text-decoration-none d-none d-md-block">❯</a>
+        </div>
+      </section>
+    </div>
+    <footer class="lato container-fluid text-dark d-flex justify-content-center pt-3"> 
+        <img src="Images/logo.png" alt="Penda logo" class="px-2 pt-1 d-md-none">
+        <p class="text-dark px-2 pt-md-3" href="#">Penda Inernational Leadership Conference. Some rights reserved &copy2023</p>
+    </footer>
```

Comment:
- [ ] I think your `footer` should have your logo aligned to the left and text aligned to the right, but for the desktop view, we can't see the logo. Could you please fix that? 🤔

### your footer 
![Screenshot from 2022-11-26 02-21-33](https://user-images.githubusercontent.com/89228743/204068705-f9043ffc-87f7-464c-8814-900f4ff34b19.png)